cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -170,7 +172,22 @@ export class Analysis extends Application {
 
     protected selectTarget(target: string[]): void {
         for (let i = 0; i < target.length; i++) {
-            cy.get("div.pf-v5-c-empty-state__content").children("h4").contains(target[i]).click();
+            if (
+                target.includes("OpenJDK 11") ||
+                target.includes("OpenJDK 17") ||
+                target.includes("OpenJDK 21")
+            ) {
+                click(openjdkToggleButton);
+                clickByText(dropDownMenu, target[i]);
+            } else if (target.includes("camel:3") || target.includes("camel:4")) {
```

Comment:
We have to use `targets[i].includes` as targets is an array we're iterating.
And it could be just `if (target[i].includes('camel'))`